python/py-pkginfo/Portfile

```diff
@@ -20,8 +20,8 @@ long_description \
 
 homepage            https://pypi.python.org/pypi/pkginfo
 
-checksums           rmd160  2b2f25fa0f12b0df61f2c64ea0a289d6640f650e \
-                    sha256  c6bc916b8298d159e31f2c216e35ee5b86da7da18874f879798d0a1983537c86 \
-                    size    450821
+checksums           rmd160  2c27cb7f77179ea45f52c814dc5e61c20d53d56a \
+                    sha256  8ad91a0445a036782b9366ef8b8c2c50291f83a553478ba8580c73d3215700cf \
+                    size    451375
 
-python.versions     38 39 310 311 312 313
+python.versions     39 310 311 312 313
```

Comment:
it seems to me there are still dependents of the py38 subport so you cannot just remove it.